### ISSUE TITLE
[Enhancement] Search memory for cheats

### ISSUE THREAD

**Issue by branchus** [nba-emu/NanoBoyAdvance#300]
Hey Mireille

Really love NanoBoyAdvance, just a suggestion. I don't like to use existing cheats code found online, but sometimes I do want to search for a certain thing and cheat on that. 

VBA has this function and works really well. below is a screenshot I found online, but it tells all. After loading the game, can use this window to search a number in the game, then back to game and make sure the number changed, then search again, generally after repeat this for 2 to 3 time, it will be able to find the memory address then can change the value in the memory.

This function doesn't relate to any of the accuracy and not game critical, so totally fine if this is not your interest.

Thanks

![875-lfwmvqlmwb](https://user-images.githubusercontent.com/1289143/236609583-914cfdbb-15b0-4c1c-af31-96e3a0823176.png)

**Comment by fleroviux** [comment]
Yeah, I recognize that something like this could be useful. It's quite far back in my TODO list for now, after debugging tools and cheats and things like that.

**Comment by legendarysnake0** [comment]
I hope its available on 1.8, playing j-rpgs on mgba because of cheats its anoying because the graphics are way better on nanoboy.

**Comment by branchus** [comment]
> I hope its available on 1.8, playing j-rpgs on mgba because of cheats its anoying because the graphics are way better on nanoboy.

You know the save are compatible, so you can use vba-m or mgba to cheat, save, then switch back to NBA :)
